### PR TITLE
feat: improve SDKs and rest support

### PR DESCRIPTION
**Pull request by zifeo**

### CODE REVIEW

File: typegate/tests/runtimes/rest/rest.py
Comment (Natoandro) [inline typegate/tests/runtimes/rest/rest.py:1]: I think we should rename this into `http` because the runtime name is actually `HTTPRuntime`.

**Review by Natoandro — APPROVED** [review]
:+1:

File: typegate/tests/runtimes/rest/rest.py
Comment (zifeo) [reply typegate/tests/runtimes/rest/rest.py:1]: Totally!

**Review by Natoandro — APPROVED** [review]

File: typegraph/core/src/lib.rs [hunk 1]
```diff
@@ -28,16 +28,48 @@ pub mod wit {
...
+            Ok(())
+        }
+    }
 }
 
 pub struct Lib {}
 
 impl wit::core::Core for Lib {
     fn init_typegraph(params: TypegraphInitParams) -> Result<()> {
+        /*
```
Comment (Natoandro) [inline typegraph/core/src/lib.rs:64]: Uh?

**Pull request merged**